File: Kotlin/Swap/swap.kt
fun swap( x : Int, y : Int ) {
	println("x: $x")
	println("y: $y")

	var tmp : Int = x
	x = y
	y = tmp

        println("x: $x")
        println("y: $y")
}

fun main( args : Array<String> ) {
	swap( 3, 5 )
}

File: C#/Swap/Swap.cs
using System;

public class Program{

	public static void Swap(ref int x, ref int y){
		int tmp = x;
		x = y;
		y = tmp;
	}

	public static void Main(string[] args){
		int x = 5; y = 3;
		Swap( ref x, ref y);

	        System.Console.WriteLine(x);
		System.Console.WriteLine(y);
	
	}
}
